### ISSUE TITLE
API integration with Eventbrite

### ISSUE THREAD

**Issue by asgrim** [phphants/phph-site#106]
Create API integration with Eventbrite. Depends on #105.

When a meetup has been created, add a button to "create eventbrite registration".

This should use the data entered to automatically create an Eventbrite form using our standard structure and details.

The only data we don't currently store is the speaker photo, so that will be part of this.

**Comment by asgrim** [comment]
Probably not necessary as we'll be adding attendance records internally in #113. Closing this, going to create a separate task to REMOVE Eventbrite integration.
